content/docs/guides/gke/troubleshooting-gke.md

```diff
@@ -24,6 +24,64 @@ Here are some tips for troubleshooting Cloud IAP.
 
  * Make sure you are using HTTPS
 
+### DNS name not registered
+
+This section provides troubleshooting information for problems creating a DNS entry for your ingress. This section assumes
+you are using Cloud Endpoints and the DNS name
```

Comment:
Add a URL for "Cloud Endpoints" to give the reader an idea what we're talking about:
https://cloud.google.com/endpoints/